### PR TITLE
Change severity to level

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-cosmos-cmd-tlm-api/app/models/messages_api.rb [hunk 1]
```diff
@@ -16,11 +16,31 @@
 # This file may also be used under the terms of a commercial license
 # if purchased from OpenC3, Inc.
 
-require_relative 'messages_thread'
+require_relative "messages_thread"
 
 class MessagesApi
-  def initialize(uuid, channel, history_count = 0, start_offset: nil, start_time: nil, end_time: nil, types: nil, severity: nil, scope:)
-    @thread = MessagesThread.new(channel, history_count, start_offset: start_offset, start_time: start_time, end_time: end_time, types: types, severity: severity, scope: scope)
```
Comment (jmthomas) [inline openc3-cosmos-cmd-tlm-api/app/models/messages_api.rb:23]: I reformatted this using prettier with ruby. Let me know what you think.

File: openc3/lib/openc3/logs/text_log_writer.rb [hunk 2]
```diff
@@ -79,7 +79,7 @@ def close_file(take_mutex = true)
         # Need to write the OFFSET_MARKER for each packet
         @last_offsets.each do |redis_topic, last_offset|
           time = Time.now
-          data = { time: time.to_nsec_from_epoch, '@timestamp' => time.xmlschema(3), severity: 'INFO', "microservice_name" => Logger.microservice_name, "container_name" => @container_name, "last_offset" => last_offset, "redis_topic" => redis_topic, "type" => "offset" }
+          data = { time: time.to_nsec_from_epoch, '@timestamp' => time.xmlschema(3), level: 'INFO', "microservice_name" => Logger.microservice_name, "container_name" => @container_name, "last_offset" => last_offset, "redis_topic" => redis_topic, "type" => "offset" }
```
Comment (jmthomas) [inline openc3/lib/openc3/logs/text_log_writer.rb:82]: This is a change to the output file but it matches our current implementation

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/icons/index.js [hunk 3]
```diff
@@ -78,8 +78,8 @@ const UnknownToAstroStatus = {
   FATAL: 'fatal',
   error: 'critical',
   ERROR: 'critical',
-  warn: 'serious',
-  WARN: 'serious',
+  warn: 'caution',
+  WARN: 'caution',
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/icons/index.js:82]: Make the yellow limits yellow and not orange

File: openc3-cosmos-cmd-tlm-api/app/models/messages_thread.rb [hunk 4]
```diff
@@ -50,19 +56,21 @@ def setup_thread_body
     if @start_time
       # start_time can be at most 1 minute in the future to prevent
       # spinning up threads that just block forever
-      if (@start_time - ALLOWABLE_START_TIME_OFFSET_NSEC) > Time.now.to_nsec_from_epoch
+      if (@start_time - ALLOWABLE_START_TIME_OFFSET_NSEC) >
+           Time.now.to_nsec_from_epoch
```
Comment (ryanmelt) [inline openc3-cosmos-cmd-tlm-api/app/models/messages_thread.rb:60]: This is dumb.  How wide is it configured to break into two lines?

File: openc3-cosmos-cmd-tlm-api/app/models/messages_thread.rb [hunk 5]
```diff
@@ -186,19 +200,19 @@ def handle_log_entry(log_entry)
...
+      case level
+      when "DEBUG"
+        return nil if @level != "DEBUG"
+      when "INFO"
+        return nil if @level == "WARN" or @level == "ERROR" or @level == "FATAL"
+      when "WARN"
+        return nil if @level == "ERROR" or @level == "FATAL"
+      when "ERROR"
+        return nil if @level == "FATAL"
+      else
```
Comment (ryanmelt) [inline openc3-cosmos-cmd-tlm-api/app/models/messages_thread.rb:215]: This is a bug.   if severity was FATAL everything would be filtered.

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**